source/config/Config.cpp

```diff
@@ -121,6 +121,10 @@ bool PlainConfig::LoadFromJson(const Crt::JsonView &json)
             auto path = FileUtils::ExtractExpandedPath(json.GetString(jsonKey).c_str());
             if (FileUtils::FileExists(path))
             {
+                FileUtils::ValidateFilePermissions(
```

Comment:
Why are we validating over here if we have the same code added in `Validate` method?